packages/create-react-admin/src/cli.tsx

```diff
@@ -49,6 +54,16 @@ const cli = meow(
 
 if (cli.flags.h) {
     cli.showHelp();
+} else if (cli.flags.basic) {
+    render(
+        <App
+            name={cli.input.length > 0 ? cli.input[0] : undefined}
+            dataProvider="none"
+            authProvider="none"
+            resources={[]}
+            install="skip"
+        />
+    );
```

Comment:
You can avoid an `else if` by declaring local `dataProvider`, `authProvider`, `resource` and `skip` variables, initialize them depending on the `cli.flags` and then render the `App`